api/src/main/java/ai/djl/modality/cv/translator/SemanticSegmentationTranslator.java

```diff
@@ -110,10 +110,12 @@ public Image processOutput(TranslatorContext ctx, NDList list) {
             }
             int originW = (int) ctx.getAttachment("originalWidth");
             int originH = (int) ctx.getAttachment("originalHeight");
-            NDArray intRet = manager.create(bb, new Shape(height, width, CHANNEL), DataType.UINT8);
-            NDArray resized = NDImageUtils.resize(intRet, originW, originH);
+            NDArray fullImage =
+                    manager.create(bb, new Shape(height, width, CHANNEL), DataType.UINT8);
+            NDArray resized = NDImageUtils.resize(fullImage, originW, originH);
```

Comment:
Does image change data type after resizing?

Reply:
When calling from DJL repo, no. When calling from DJL-demo where the app is, yes. On the resize, DJL-Demo calls BitmapImageFactory (instead of BufferedImageFactory which is called by DJL) and this ends up changing the data type so this is a safeguard that prevents the datatype from changing before it is sent to be returned, where it must be of type `UINT8`.

Reply:
I would be more inclined to treat that as a problem with resize. Maybe add a `toType` to the `resize` method that is being called to ensure that it doesn't change data types

Reply:
Will push change in new pr